src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -729,7 +738,7 @@ public static ComponentResultEntity toEntity(LoadFlowResult.ComponentResult comp
 
         Mono<UUID> networkUuid = getNetworkUuid(studyName, userId);
 
-        return networkUuid.flatMap(uuid -> {
+        Mono<UUID> resultUuid = networkUuid.flatMap(uuid -> {
```

Comment:
Keep the previous style, don't create a new variable and just chain your flatmap to this returned expression

Reply:
Corrected